include/stdexec/execution.hpp

```diff
@@ -413,6 +413,13 @@ namespace stdexec {
   using __as_awaitable::as_awaitable_t;
   extern const as_awaitable_t as_awaitable;
 
+  template <class _Fn, class... _Args>
+  struct _ERRONEOUS_SUBSTITUTION_ : std::false_type {
+    friend auto operator,(const _ERRONEOUS_SUBSTITUTION_&, auto) -> _ERRONEOUS_SUBSTITUTION_ {
+      return {};
+    }
```

Comment:
It occurs to me that we can save some template instantiations here because this friend function doesn't need to be a template. See https://godbolt.org/z/M9599dqM4